invenio_i18n/config.py

```diff
@@ -49,3 +49,5 @@
 It is used only when the login manager is installed and a user is
 authenticated. Set to ``None`` to prevent selector from being used.
 """
+
+BABEL_DOMAIN = "messages"
```

Comment:
Did you add this because you wanted to expose this config var?
The default is already `messages`.

Reply:
i rethought the whole thing. I was wrong. [commit](https://github.com/inveniosoftware/invenio-i18n/pull/108/commits/7e17fdf41ce31c3ee4034c61b66260c689a5ac8e) fixes this. So to summarize `BABEL_DOMAIN` isn't necessary.